### PR TITLE
Add Tizen 9.0 M2 Release Notes

### PR DESCRIPTION
**Pull request by seokwon-jung**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+
+### Network and connectivity
+
+#### New and changed features
+
+- Network
+  - New APIs for monitoring and retrieving the state of default gateway and DNS have been added.
+  - New API for initiating a one-time online HTTP check has been added.
+- Wi-Fi Aware
+  - New set of APIs to manage Wi-Fi Aware functionalities has been added.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:481]:
```suggestion
  - New set of APIs to manage Wi-Fi Aware functionalities have been added.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 2]
```diff
@@ -0,0 +1,663 @@
...
+    - New main loop model to provide improved functionality has been added.
+  - Tizen.Network.Connection
+    - WPA3 security type has been added.
+  - TizenWebView has been removed.
+  - Tizen.NUI.Wearable has been deprecated.
+
+
+### Machine learning
+
+## New and changed features
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:634]:
```suggestion
#### New and changed features
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 3]
```diff
@@ -0,0 +1,663 @@
...
+
+### Machine Learning (ML) API updates
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:636]:
```suggestion
#### Machine Learning (ML) API updates
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 4]
```diff
@@ -0,0 +1,663 @@
...
+  -  ML.Service API Updates
+    - ML Service API supports application developers to use AI offloading service using various protocols such as TCP, MQTT, and other custom connection via NNStreamer-Edge.
+  - Add NNFW enum type to support TensorRT and QNN.
+
+### NNStreamer and Related modules update
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:641]:
```suggestion
#### NNStreamer and Related modules update
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 5]
```diff
@@ -0,0 +1,663 @@
...
+  - NNStreamer-Edge has been upgraded to version 0.2.6.
+    - Support custom connection (user defined protocol).
+    - Support Multi-Connectivity Framework (MCF) using custom connection.
+
+  - MLOps-Agent has been upgraded to version 1.8.6.
+    - Parse the path for AI model or recource files from installed RPK.
+
+  - Support ML computation offloading to an Android device (Experimental).
+
+## Fixes
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:659]:
```suggestion
#### Fixes
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 6]
```diff
@@ -0,0 +1,663 @@
...
+  - Reported bugs in NNStreamer and ML API have been fixed.
+
+## Known issues
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:662]:
```suggestion
#### Known issues
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 7]
```diff
@@ -0,0 +1,663 @@
...
+    - Add new inference filter to support the ONNX Runtime.
+    - Add new inference filter to support the SNPE version 2 (Qualcomm Neural Processing SDK).
+    - Add new inference filter to support the QNN (Qualcomm AI Engine Direct).
+    - Add new inference filter to support the TensorRT version 10.
+    - Add new inference filter to support the ncnn engine (Tencent).
+    - Add new inference filter to support the ExecuTorch (Experimental).
+
+  - NNStreamer-Edge has been upgraded to version 0.2.6.
+    - Support custom connection (user defined protocol).
+    - Support Multi-Connectivity Framework (MCF) using custom connection.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:652]:
```suggestion
    - Support Multi Connectivity Framework (MCF) using custom connection.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 8]
```diff
@@ -0,0 +1,663 @@
...
+  - Support for creating xpath when processes name are same.
+  - Support for updating function of application information.
+  - Support for updating function of XPath information.
+  - Support for changing the port number(50051-50060)
+  - Support for separating event registration.
+  - Improve the performance by refactoring processing event.
+  - Support for navigation functions.
+  - Support for getting the parent.
+  - Support for aurum-python to improve user convienent.
+- UI Analyzer (internal Tool)
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:374]:
```suggestion
- UI Analyzer (Internal Tool)
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 9]
```diff
@@ -0,0 +1,663 @@
...
+- Email
+  - Potential defects have been fixed.
+
+### Web framework
+
+#### New and changed features
+
+- WebAuthn has been supported based on Tizen Native WebAuthn.
+- CSS nesting has been supported.
+- LLVM toolchain has been upgraded to version 18.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:529]:
```suggestion
- LLVM Toolchain has been upgraded to version 18.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 10]
```diff
@@ -0,0 +1,663 @@
...
+  - MLOps-Agent has been upgraded to version 1.8.6.
+    - Parse the path for AI model or recource files from installed RPK.
+
+  - Support ML computation offloading to an Android device (Experimental).
+
+## Fixes
+  - Reported bugs in NNStreamer and ML API have been fixed.
+
+## Known issues
+  - XNNPACK delegate of TensorFlow-Lite2 is temporarily disabled due to the toolchain version issue.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:663]:
```suggestion
  - XNNPACK delegate of TensorFlow-Lite2 is temporarily disabled due to the Toolchain version issue.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 11]
```diff
@@ -0,0 +1,663 @@
...
+
+- Fix the autofill related crash/freeze issues
+- Fix the memory leak issue of SkiaOutPutDeivceOffscreen
+- Fix the Fullscreen API crash in webapps
+- Replace the deprecated CHROMIUM_image with CHROMIUM_shared_image
+- Fix the RWI websocket disconnect issue
+- Fix to support WebM container
+
+
+### Lightweight web solution
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:541]:
```suggestion
### Lightweight Web Solution
```

File: docs/platform/release-notes/tizen-9-0-m2.md (hunk 6 above)
Comment (maliha-srbd) [reply docs/platform/release-notes/tizen-9-0-m2.md:662]: @seokwon-jung Please confirm if this is okay. I have considered that "Known issues" fall under the "### Machine Learning" (third-level heading)

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 12]
```diff
@@ -0,0 +1,663 @@
...
+  - New API for initiating a one-time online HTTP check has been added.
+- Wi-Fi Aware
+  - New set of APIs to manage Wi-Fi Aware functionalities has been added.
+    - Allows applications to create, configure, and manage Wi-Fi Aware sessions.
+    - Supports various session types such as Publish and Subscribe.
+    - Provides mechanisms for sending and receiving messages between peers.
+  - Includes features for enabling and disabling Wi-Fi Aware, handling session termination, and discovering nearby services.
+  - Enhanced capabilities for establishing secure data paths and managing peer connections.
+- Bluetooth
+  - New disconnection error type to handle MIC failure bas been added.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:488]:
```suggestion
  - New disconnection error type to handle MIC failure has been added.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 13]
```diff
@@ -0,0 +1,663 @@
...
+
+#### New and changed features
+
+- Open Source
+  - llvm version 10.0.0 has been upgraded to version 17.0.6.
+  - linux-glibc-devel version 3.10 has been upgraded to version 5.4.
+  - libgc version 8.0.4 has been upgraded to version 8.2.6.
+
+
+### System (base)
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:80]:
```suggestion
### System (Base)
```

**Review by maliha-srbd — COMMENTED** [review]
Hello @seokwon-jung , I hope you're doing well. Please check the suggested changes and incorporate them accordingly.

**Comment by seokwon-jung** [comment]
I updated it , please review again
thanks in advance

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 14]
```diff
@@ -0,0 +1,663 @@
...
+  - Support 2D Panel to draw 2D UI Content inside 3D Scene.
+  - Support Capture for SceneView.
+- Drag and Drop
+  - Support for multiple windows on a single process has been added.
+- Performance/Memory Improvement
+  - Optimize render thread bottleneck.
+  - Optimize Property update logic in render thread.
+  - Optimize to control Vector capacity release for memory usage.
+  - Support custom shader pre-compile option.
+  - Support to synchronize rendering fps with lottie's fps.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:361]:
```suggestion
  - Support to synchronize rendering FPS with Lottie's FPS.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 15]
```diff
@@ -0,0 +1,663 @@
...
+- Aurum
+  - Improve the performance of finding element and dump objects by adding new methods.
+  - Support for sorting child objects by coordinates.
+  - Support for creating xpath when processes name are same.
+  - Support for updating function of application information.
+  - Support for updating function of XPath information.
+  - Support for changing the port number(50051-50060)
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:368]:
```suggestion
  - Support for changing the port number (50051-50060)
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 16]
```diff
@@ -0,0 +1,663 @@
...
+  - Support for separating event registration.
+  - Improve the performance by refactoring processing event.
+  - Support for navigation functions.
+  - Support for getting the parent.
+  - Support for aurum-python to improve user convienent.
+- UI Analyzer (internal Tool)
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:374]:
```suggestion
- UI Analyzer (Internal Tool)
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 17]
```diff
@@ -0,0 +1,663 @@
...
+  - Support for description information
+  - Support for the count of objects in object tree.
+  - Support for vertical scrollbar in screen shot
+  - Support for aurum-python
+  - Support for the distiction between several root layers
+  - Improve the performance when window is resized.
+  - Support for player cancel function.
+  - Support for upgrade the vulnerable.
+  - Support for waiting the grpc server ready.
+  - Support for role, highlightable attributes.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:386]:
```suggestion
  - Support for role and highlightable attributes.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 18]
```diff
@@ -0,0 +1,663 @@
...
+  - Support for change the sdb port.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:387]:
```suggestion
  - Support for change the SDB port.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 19]
```diff
@@ -0,0 +1,663 @@
...
+  - Improve the offscreen rendering performance of multiple shapes in lottie-player.
+  - A trim path dynamic property in lottie-player has been added.
+- Clipboard
+  - Support for multi-type copy and paste.
+  - A HasType in Clipboard has been added.
+
+#### Fixes
+
+  - Fix bug when find element with showing condition.
+  - Fix Memory leaks
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:400]:
```suggestion
  - Fix memory leaks
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 20]
```diff
@@ -0,0 +1,663 @@
...
+  - Fix parameter for catch
+  - Fix the bug of ordering object.
+  - Fix Dialog and AlertDialog behaviours for Accessibility.
+  - Fix View to use control-accessible State::SENSITIVE for Accessibility.
+  - Fix Navigator to support Page with Transitions.
+  - Fix Various Accessibility defects.
+  - Fix Various LottieAnimation defects.
+  - Fix Various WidgetView defects.
+  - Fix the bug during touch event hittest.
+  - Fix svg image loading defects.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:415]:
```suggestion
  - Fix SVG image loading defects.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 21]
```diff
@@ -0,0 +1,663 @@
...
+  - Deprecated legacy face, image, surveillance APIs.
+  - Deprecated inference APIs.
+  - New machine learning task APIs have been added.
+- Single Oriented AI Service framework (Internal)
+  - Support AutoZoom Service API
+  - Provide adaptation interfaces to support various backends such as various inputs and machine learning frameworks
+    - Support MediaVision framework as a machine learning based vision backend
+    - Support Camera API, OpenCV and vision source as Camera input backends
+- Media Streamer
+  - Media Streamer APIs has been removed.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:458]:
```suggestion
  - Media Streamer APIs have been removed.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 22]
```diff
@@ -0,0 +1,663 @@
...
+- Native WebRTC
+  - Renegotiation has been supported.
+  - Domain name as connection address of ICE candidate has been supported.
+  - Consent-freshness of RFC7675 has been supported.
+  - New enum for 'max-compat' bundle policy has been added.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:463]:
```suggestion
  - New enum for ``max-compat`` bundle policy has been added.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 23]
```diff
@@ -0,0 +1,663 @@
...
+
+### Network and connectivity
+
+#### New and changed features
+
+- Network
+  - New APIs for monitoring and retrieving the state of default gateway and DNS have been added.
+  - New API for initiating a one-time online HTTP check has been added.
+- Wi-Fi Aware
+  - New set of APIs to manage Wi-Fi Aware functionalities has been added.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:481]:
```suggestion
  - New set of APIs to manage Wi-Fi Aware functionalities have been added.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 24]
```diff
@@ -0,0 +1,663 @@
...
+
+### Security
+
+#### New and changed features
+
+- PQC Algorithm Support
+  - The ligoqs has been added for PQC algorithms.
+  - The key manager supports ML-KEM.
+- Passkey Support
+  - New native and C# WebAuthn APIs are provided.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:510]:
```suggestion
  - New Native and C# WebAuthn APIs are provided.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 25]
```diff
@@ -0,0 +1,663 @@
...
+  - Tizen.Messaging.Messages has been deprecated.
+  - Tizen.Applications.ComponentBased
+    - TimeZoneChanged event handler has been added.
+  - Tizen.Applications.Common
+    - TimeZoneEvent has been added.
+  - Tizen.Applications.AttachPanel has been deprecated.
+  - Tizen.Multimedia.Vision
+    - InferenceTargetType and InferenceBackendType have been removed.
+  - Tizen.Multimedia.Util
+    - Deprecated APIs has been removed.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:611]:
```suggestion
    - Deprecated APIs have been removed.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 26]
```diff
@@ -0,0 +1,663 @@
...
+    - New main loop model to provide improved functionality has been added.
+  - Tizen.Network.Connection
+    - WPA3 security type has been added.
+  - TizenWebView has been removed.
+  - Tizen.NUI.Wearable has been deprecated.
+
+
+### Machine learning
+
+## New and changed features
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:634]:
```suggestion
#### New and changed features
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 27]
```diff
@@ -0,0 +1,663 @@
...
+
+### Machine Learning (ML) API updates
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:636]:
```suggestion
#### Machine Learning (ML) API updates
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 28]
```diff
@@ -0,0 +1,663 @@
...
+  -  ML.Service API Updates
+    - ML Service API supports application developers to use AI offloading service using various protocols such as TCP, MQTT, and other custom connection via NNStreamer-Edge.
+  - Add NNFW enum type to support TensorRT and QNN.
+
+### NNStreamer and Related modules update
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:641]:
```suggestion
#### NNStreamer and Related modules update
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 29]
```diff
@@ -0,0 +1,663 @@
...
+    - Add new inference filter to support the ONNX Runtime.
+    - Add new inference filter to support the SNPE version 2 (Qualcomm Neural Processing SDK).
+    - Add new inference filter to support the QNN (Qualcomm AI Engine Direct).
+    - Add new inference filter to support the TensorRT version 10.
+    - Add new inference filter to support the ncnn engine (Tencent).
+    - Add new inference filter to support the ExecuTorch (Experimental).
+
+  - NNStreamer-Edge has been upgraded to version 0.2.6.
+    - Support custom connection (user defined protocol).
+    - Support Multi-Connectivity Framework (MCF) using custom connection.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:652]:
```suggestion
    - Support Multi Connectivity Framework (MCF) using custom connection.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 30]
```diff
@@ -0,0 +1,663 @@
...
+
+  - MLOps-Agent has been upgraded to version 1.8.6.
+    - Parse the path for AI model or recource files from installed RPK.
+
+  - Support ML computation offloading to an Android device (Experimental).
+
+## Fixes
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:659]:
```suggestion
#### Fixes
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 31]
```diff
@@ -0,0 +1,663 @@
...
+  - Reported bugs in NNStreamer and ML API have been fixed.
+
+## Known issues
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:662]:
```suggestion
#### Known issues
```

Comment (maliha-srbd) [reply docs/platform/release-notes/tizen-9-0-m2.md:662]: Please confirm if this is okay. I have considered that the should "Known issues" fall under the "### Machine Learning" (third-level heading) category. @seokwon-jung

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 32]
```diff
@@ -0,0 +1,663 @@
...
+  - Improve the performance when window is resized.
+  - Support for player cancel function.
+  - Support for upgrade the vulnerable.
+  - Support for waiting the grpc server ready.
+  - Support for role and highlightable attributes.
+  - Support for change the SDB port.
+- HandDrawingEngine
+  - Support for drawing engine based on vector graphics has been added.
+- Vector Graphics
+  - Improve the offscreen rendering performance of multiple shapes in lottie-player.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:391]:
```suggestion
  - Improve the offscreen rendering performance of multiple shapes in Lottie-player.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 33]
```diff
@@ -0,0 +1,663 @@
...
+  - A trim path dynamic property in lottie-player has been added.
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:392]:
```suggestion
  - A trim path dynamic property in Lottie-player has been added.
```

File: docs/platform/release-notes/tizen-9-0-m2.md [hunk 34]
```diff
@@ -0,0 +1,663 @@
...
+  - Optimize to control Vector capacity release for memory usage.
+  - Support custom shader pre-compile option.
+  - Support to synchronize rendering FPS with Lottie's FPS.
+- Aurum
+  - Improve the performance of finding element and dump objects by adding new methods.
+  - Support for sorting child objects by coordinates.
+  - Support for creating xpath when processes name are same.
+  - Support for updating function of application information.
+  - Support for updating function of XPath information.
+  - Support for changing the port number (50051-50060)
```
Comment (maliha-srbd) [inline docs/platform/release-notes/tizen-9-0-m2.md:368]:
```suggestion
  - Support for changing the port number (50051-50060).
```

**Review by maliha-srbd — COMMENTED** [review]
I have suggested 3 more comments, please incorporate them. @seokwon-jung

**Review by maliha-srbd — APPROVED** [review]
PR LGTM

**Pull request merged**